infrastructure/terraform/components/iam/module_amplify_branch.tf

```diff
@@ -1,7 +1,7 @@
 module "amplify_branch" {
   source = "../../modules/amp_branch"
 
-  name           = var.environment
+  name           = "main"
```

Comment:
just checking if this will always be main, hence mentioned directly here? or does this need to come from a tfvars? Same with line 15 here :)

Reply:
yea i think it should always be main, as its the base deployed branch, all other branches in the app use the branch component. I found my dev build was trying to use a dev branch which didnt exist before this change